configure.ac

```diff
@@ -1,225 +1,175 @@
-# wavpack 5.4.0 configure.ac
+dnl wavpack 5.4.0 configure.ac
 
-AC_INIT(wavpack, 5.4.0, bryant@wavpack.com)
+AC_INIT([wavpack], [5.4.0], [bryant@wavpack.com])
 AC_CONFIG_SRCDIR([src/pack.c])
-AM_INIT_AUTOMAKE([serial-tests])
+AM_INIT_AUTOMAKE([-Wall 1.16 serial-tests subdir-objects foreign no-dist-gzip dist-xz])
 AM_MAINTAINER_MODE
 
 LIBWAVPACK_MAJOR=5
 LIBWAVPACK_MINOR=4
 LIBWAVPACK_MICRO=0
-LIBWAVPACK_VERSION_STRING=$LIBWAVPACK_MAJOR.$LIBWAVPACK_MINOR.$LIBWAVPACK_MICRO
-
-AC_DEFINE_UNQUOTED(LIBWAVPACK_MAJOR, $LIBWAVPACK_MAJOR, [libwavpack major version])
-AC_DEFINE_UNQUOTED(LIBWAVPACK_MINOR, $LIBWAVPACK_MINOR, [libwavpack minor version])
-AC_DEFINE_UNQUOTED(LIBWAVPACK_MICRO, $LIBWAVPACK_MICRO, [libwavpack micro version])
-AC_DEFINE_UNQUOTED(LIBWAVPACK_VERSION_STRING, "$LIBWAVPACK_VERSION_STRING", [libwavpack version string])
-
-# set libtool versioning
-# +1 :  0 : +1   == new interface that does not break old one.
-# +1 :  0 :  0   == changed/removed an interface. Breaks old apps.
-#  ? : +1 :  ?   == internal changes that doesn't break anything.
-# CURRENT : REVISION : AGE
+LIBWAVPACK_VERSION_STRING="${LIBWAVPACK_MAJOR}.${LIBWAVPACK_MINOR}.${LIBWAVPACK_MICRO}"
+
+AC_DEFINE_UNQUOTED([LIBWAVPACK_MAJOR], [${LIBWAVPACK_MAJOR}], [libwavpack major version])
+AC_DEFINE_UNQUOTED([LIBWAVPACK_MINOR], [${LIBWAVPACK_MINOR}], [libwavpack minor version])
+AC_DEFINE_UNQUOTED([LIBWAVPACK_MICRO], [${LIBWAVPACK_MICRO}], [libwavpack micro version])
+AC_DEFINE_UNQUOTED([LIBWAVPACK_VERSION_STRING], ["${LIBWAVPACK_VERSION_STRING}"], [libwavpack version string])
+
+dnl set libtool versioning
+dnl +1 :  0 : +1   == new interface that does not break old one.
+dnl +1 :  0 :  0   == changed/removed an interface. Breaks old apps.
+dnl  ? : +1 :  ?   == internal changes that doesn't break anything.
+dnl CURRENT : REVISION : AGE
 LT_CURRENT=3
 LT_REVISION=3
 LT_AGE=2
 
-AC_SUBST(LT_CURRENT)
-AC_SUBST(LT_REVISION)
-AC_SUBST(LT_AGE)
+AC_SUBST([LT_CURRENT])
+AC_SUBST([LT_REVISION])
+AC_SUBST([LT_AGE])
 
-# Check for os version
-VERSION_OS=$(uname -s)
```

Comment:
this is not portable to cross-compilation, because it will implicitly use the `CBUILD` os and not the `CHOST` one